src/components/TextInput/TextInput.tsx

```diff
@@ -49,10 +48,11 @@ export const TextInput = forwardRef(
       name,
       status = 'info',
       type = 'text',
+      maxLength,
```

Comment:
Is the goal here to avoid propagating `maxLength` to the `input`?

Reply:
This is remnant of a prior fix attempt, good catch.